release-notes/opensearch-release-notes-2.10.0.md

```diff
@@ -0,0 +1,671 @@
+OpenSearch and OpenSearch Dashboards 2.10.0 Release Notes
+
+## Release Highlights
+
+### New Features
+
+### Experimental Features
```

Comment:
@sohami The experimental features related to concurrent search should be added here.

Reply:
@jhmcintyre Please let us know when you have release highlights ready. Thank you!